### ISSUE TITLE
Gather basic system facts as part of the plan execution.

### ISSUE THREAD

**Issue by sudharsh** [apigee/henchman#26]

**Comment by jlin21** [comment]
What are some basic system facts?

**Comment by jlin21** [comment]
http://docs.ansible.com/ansible/playbooks_variables.html#information-discovered-from-systems-facts

**Comment by jlin21** [comment]
We could make a script (similar to modules) that grabs facts.  It could be deployed during plan setup

**Comment by sudharsh** [comment]
yep, pre-modules transfer. This should also ideally plug in with facter. Let's start with some basic facts like uname, time, network interfaces, kernel versions. etc. We'll add more as and when required.
